### PR TITLE
Add API to my Leaderboard project

### PR DESCRIPTION
**Pull request by edwardoluyara**
- This is a project to store asynchronous data from an API

### CODE REVIEW

**Review by oloomoses — COMMENTED** [review]
Hi @oluyaratosin123, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a readme file. Kindly add a descriptive readme as per the general requirements in the project description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/API.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const sendScore = () => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/JISCZSyYfIzlC6N08Rja/scores/', {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: nameInput.value, score: scoreInput.value }),
+  });
+};
```
Comment (VanessaAoki) [inline src/API.js]: - [ ] Great job implementing the fetch leaderboard, I can see all the submitted data 👏  but right now your post is not working. When I changed the parameter from `name` to `user`, it worked! you should consider double-checking it. You can take a look at it here:

image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Usage: N/A
+Run tests: Run syntax tests with the use of the npx commands given in the linters.yml file.
+Deployment: Run index.html on your local server.
+
+# Live Version
+Live Demo Link: 
+# Authors
+
+GitHub: @oluyaratosin123
+LinkedIn: Oluyara Edward
```
Comment (VanessaAoki) [inline README.md]: - [ ] [OPTIONAL]: To keep your repo even more professional, you should consider adding the actual links to your social media.

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @oluyaratosin123, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**